test/e2e/parallel_test.go

```diff
@@ -1,4 +1,5 @@
-// +build e2e
+// +build skip
+// Skip test for now. Change back to "e2e" when fixed
```

Comment:
Why not add a `t.Skip("SRVKE-XXX: Skipping flaky test for now")` in the test itself? :thinking:

Reply:
Because I didn't know it better :)

Going to fix that now.